pom.xml

```diff
@@ -228,7 +228,7 @@
         <rhinoVersion>1.7R4</rhinoVersion>
         <rhinoJsr223Version>1.1</rhinoJsr223Version>
         <saajVersion>1.3</saajVersion>
-        <saxonVersion>9.6.0-1</saxonVersion>
+        <saxonVersion>9.6.0-7</saxonVersion>
```

Comment:
9.7.0-1 is available, which also includes support for XPath 3.1. Why don't you go to the latest path version instead of upgrading to the latest minor?